web/styles/settings.css

```diff
@@ -602,8 +602,8 @@ input[type="checkbox"] {
 }
 
 #add-custom-profile-field-btn {
-    float: right;
     margin-top: 4px;
+    margin-bottom: 4px;
```

Comment:
Why this has the bottom margin for 4px and 12px? Can we keep the gap same between the button or search input and the top of the table same for all the panels?

Reply:
This is now also 12px as it is for all the other settings panel filters